GithubJobApp/GithubApiManager.swift

```diff
@@ -10,60 +10,44 @@ import Foundation
 
 struct GithubApiManager {
     
-    let jobsUrl: String = "https://jobs.github.com/positions.json?description=swift"
+    private let jobsUrl: String = "https://jobs.github.com/positions.json?description=swift"
     
-    // Dont need these vars
-//    var jobTitles = [String]()
-//    var jobCompanies = [String]()
-    
-    func getJobs(setDataFunction: (Array<Job>) -> Void){
+    func getJobs(completion: ([Job]) -> Void) {
         if let url = NSURL(string: jobsUrl) {
-        let task = NSURLSession.sharedSession().dataTaskWithURL(url) {(data, response, error) in
-            if let actualData = data { print(NSString(data: actualData, encoding: NSUTF8StringEncoding)) }
-            
-            var jobArray = [Job]()
-            var json: Array<AnyObject>?
-            do {
-                guard let actualData = data else {
+            let task = NSURLSession.sharedSession().dataTaskWithURL(url) {(data, response, error) in
```

Comment:
Also, in the arguments for the closure ( data, response, error), the style is to use _ for unused arguments.  In this case, you're only using the `data` argument, so you might right `{ data, _, _ in ...`.  The compiler usually gives you warnings if you don't -- not sure why it doesn't in this case.  And the surrounding ()s are optional and usually dropped.